src/content/test/cognitive/guidance.tsx

```diff
@@ -0,0 +1,114 @@
+            <ul>
+                <li>learning, communication, reading, writing, or math,</li>
+                <li>
+                    ability to understand or process new or complex information and learn new skills, with a reduced ability to cope
+                    independently, and/or
+                </li>
+                <li>memory and attention or visual, language, or numerical thinking.</li>
+            </ul>
+        </React.Fragment>
+        <p>
+            Web page design & structure can make content inaccessible to people with cognitive and learning disabilities. For example,
+            people with impaired short-term memory may be unable to recall passwords, remember access codes, or have trouble remembering
+            unfamiliar iconography. Additionally, people with cognitive and learning disabilities may need more support or time to complete
+            a new process or an authentication task.
+        </p>
+        <p>
+            It is important designs are created while considering the cognitive load they require of their users, as many people may
+            struggle with cognitive fatigue when completing complex, multi-stage processes. For instance, tasks like filling out forms while
+            entering important or sensitive data correctly or just trying to locate the content or feature that they need. Designs need to
+            consistently provide support to help minimize errors and complete their task.
+        </p>
+        <Markup.Columns>
+            <Markup.Do>
+                <h3>
+                    Provide users functionality to Copy & paste passwords for authentication. (<Link.WCAG_3_3_8 />)
+                </h3>
+
+                <h3>
+                    Create systems with 2-factor authentication systems with verification codes to prevent higher cognitive recall. (
+                    <Link.WCAG_3_3_8 />)
+                </h3>
+            </Markup.Do>
+
+            <Markup.Dont>
+                <h3>
+                    Don't rely on users memorizing or transcribing a username, password, or one-time verification code. (<Link.WCAG_3_3_8 />
+                    )
+                </h3>
+
+                <h3>
+                    Don't require people to re-enter information they have already provided via other means. (<Link.WCAG_3_3_7 />)
+                </h3>
+                <ul>
+                    <li>Ensure processes do not rely on memory.</li>
+                    <li>
+                        Memory barriers stop people with cognitive disabilities from using content. This includes long passwords to log in
+                        and voice menus that involve remembering a specific number or term. Make sure there is an easier option for people
+                        who need it.
+                    </li>
+                </ul>
+
+                <h3>
+                    Don't use <Markup.Code>display:none</Markup.Code>, <Markup.Code>visibility:hidden</Markup.Code> or{' '}
```

Comment:
This feels out of place?

Reply:
This tip? or specifically something about this line? This tip is because headings are important landmarks that can help with memory, and hiding them from users is a bad practice.

Reply:
This tip, but that context makes more sense. Maybe add it?